packages/grid/_modules_/grid/hooks/features/useColumnResize.tsx

```diff
@@ -127,5 +127,7 @@ export const useColumnResize = (columnsRef: React.RefObject<HTMLDivElement>, api
     };
   }, [stopListening]);
 
-  return React.useMemo(() => ({ onMouseDown: handleMouseDown }), [handleMouseDown]) as {};
+  return React.useMemo(() => ({ onMouseDown: handleMouseDown }), [
+    handleMouseDown,
+  ]) as React.HTMLAttributes<HTMLDivElement>;
```

Comment:
Hmm that's a weird type, much bigger than the simple  `{onMouseDown: (event:any)=> void}` 
What's the goal?

Also if you just want to spread it on a div, it should work with `{onMouseDown...}`

Reply:
Ok, you won. You gonna like the next commit :).